src/kit.ts

```diff
@@ -567,7 +569,7 @@ async function collectDevBatVars(devbat: string, args: string[], major_version:
           return acc;
         }, new Map());
   if (vars.get('INCLUDE') === '') {
-    console.log(`Error running ${devbat} ${args.join(' ')}, cannot find INCLUDE`);
+    log.error(localize('script.run.error', 'Error running:\n{0}\nWith args:\n{1}, cannot find INCLUDE with {2}', batContent, args.join(' '), env));
     return;
   }
```

Comment:
Not sure how readable that message is, because it has very long parts.
I'm thinking that we still need devbat in the message, as it were before, then being followed by the args. Right now, the args are coming after the devbat content.
Then a print of the env is useful, but part of a different message phrase.